scripts/Muon/GUI/Common/grouping_table_widget/grouping_table_widget_presenter.py

```diff
@@ -14,6 +14,21 @@
 maximum_number_of_groups = 20
 
 
+class RowValid(Enum):
+    invalid_for_all_runs = 0
+    valid_for_all_runs = 2
+    valid_for_some_runs = 1
+
+
+row_colors = {RowValid.invalid_for_all_runs: (255, 0, 0), RowValid.valid_for_some_runs: (255, 255, 0),
```

Comment:
are these numbers just the colours?

Reply:
Yes these are the colours specified in RGB. I'll add a comment to clarify which of the various colour specification schemes is being used here. I have a vague memory of Aidy not wanting yellow and red for the warning error colours here. Does that ring any bells with you?

Reply:
yes it does. The colours looked ok to me. It was more the magic numbers that bothered me